h5p-bildetema/src/hooks/useDbContext.ts

```diff
@@ -0,0 +1,8 @@
+import { useContext } from "react";
+import { DBContext } from "../components/App/App";
+import { Data } from "../../../common/types/types";
+
+export const useDbContext = (): Data => {
```

Comment:
Very nitpicky, but I think the hook and context should have the same casing.

```suggestion
export const useDBContext = (): Data => {
```